### ISSUE TITLE
2.3.2: Three Flashes: Blink can overlap flash, sentence need part deleted or updated, contradiction!

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#1677]
https://www.w3.org/WAI/WCAG22/Understanding/three-flashes.html

The following sentences contradict:

> Blinking usually doesn't occur at speeds of 3 per second or more **so blink and flash do not overlap**. However, blinking can occur faster than 3 per second **so there could be an overlap**.

First it says: **so blink and flash do not overlap**

Than it says: **so there could be an overlap**

We might want to change it to:

> Blinking usually doesn't occur at speeds of 3 per second or more. However, blinking can occur faster than 3 per second **so there could be an overlap**.

**Comment by jake-abma** [comment]
or change the Note to the note of 2.3.1:

> NOTE
> The terms "blinking" and "flashing" can sometimes refer to the same content.
> 
> "Blinking" refers to content that causes a distraction problem. Blinking can be allowed for a short time as long as it stops (or can be stopped)
> "Flashing" refers to content that can trigger a seizure (if it is more than 3 per second and large and bright enough). This cannot be allowed even for a second or it could cause a seizure. And turning the flash off is also not an option since the seizure could occur faster than most users could turn it off.
> Blinking usually does not occur at speeds of 3 per second or more, but it can. If blinking occurs faster than 3 per second, it would also be considered a flash.

**Comment by bruce-usab** [comment]
Thanks, that is good wordsmithing IMHO.

**Comment by mbgower** [comment]
The change was already made.
